manuscript/03-01-First-steps-with-List-syntax.md

```diff
@@ -1,8 +1,8 @@
 ## Знайомство з синтаксисом мов родини Lisp
 
-Мова Lisp була розроблена Джоном МакКарті у 1958 році і на сьогодні є однією з найстарших мов програмування, що досі використовуються. Від Lisp походить велика кількість інших мов, які називають діалектами Lisp. Мова ClojureScript - один з таких діалектів. Код на мові Lisp пишуть із використанням структур даних самої Lisp  - списків у дужках, але Clojure(Script) додав до первинного синтаксису Lisp нові структури даних, завдяки чому код на Clojure(Script) більш приємно писати та читати. I
+Мова Lisp була розроблена Джоном МакКарті у 1958 році і на сьогодні є однією з найдавніших мов програмування, що досі використовуються. Від Lisp походить велика кількість мов, які називають діалектами Lisp. ClojureScript - один з таких діалектів. На мові Lisp програми пишуть за допомогою структур даних самої Lisp  - списків у дужках, але Clojure(Script) додала до первинного синтаксису Lisp нові структури даних, завдяки чому код на Clojure(Script) більш приємно писати та читати.
 
-У ClojureScript список, першим елементом якого є функція, використовується для виклику функції. У наступному прикладі ми викликаємо функцію додавання із троьма аргументами. Зверніть увагу на те, що у мовах родини Lisp знаком `+` позначають не оператор, а функцію. Операторів у Lisp немає, лише функції.
+У ClojureScript список, першим елементом якого є функція, використовується для виклику функції. У наступному прикладі ми викликаємо функцію додавання із трьома аргументами. Зверніть увагу на те, що у мовах родини Lisp знаком `+` позначають не оператор, а функцію. Операторів у Lisp немає - лише функції.
 
 [source, clojure]
```

Comment:
All blocks of code should be written in Markdown, not in ADOC.
Please, read [CONTRIBUTING.md](https://github.com/LambdaBooks/clojurescript-unraveled/blob/master/CONTRIBUTING.md) and fix that.